docs/dev/setup/bamboo-bitbucket-jira.rst

```diff
@@ -229,13 +237,13 @@ under ``localhost:7990``.
 
    #. Personal access token for Bamboo.
 
-      - Log in as the admin user and go to Bamboo -> Profile (top right corner) -> Personal access tokens ->
+      - Log in as the admin user and go to Bamboo → Profile (top right corner) → Personal access tokens →
```

Comment:
This arrow is used in some more places of the documentation (49 in total). I'd suggest going through them and replacing all of them

Reply:
I think I would rather open another PR about that later to keep the scope of this PR constrained to just the setup instructions.